src/components/WithForwardRef/withForwardRef.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+
+import { TextArea } from '../TextArea/TextArea';
+import { textAreaDriverFactory } from '../TextArea/TextArea.driver';
+
+import { Button } from '../Button/Button';
+import { buttonDriverFactory } from '../Button/Button.driver';
+
+describe('withForwardRef', () => {
```

Comment:
I think this test file should have a generic component used for the test.
So that we won't need to add a test for each component.